tests/scripts/all.sh

```diff
@@ -1032,17 +1032,15 @@ component_test_no_64bit_multiplication () {
 
 component_build_uecc_baremetal_cmake () {
     msg "build: uecc baremetal native, cmake"
-    scripts/config.pl baremetal
     scripts/config.pl set MBEDTLS_USE_UECC
     CC=gcc cmake .
     make
 }
 
 component_build_uecc_baremetal_make () {
```

Comment:
Shall we rename the component now that it doesn't use the baremetal config anymore?

Reply:
Done